src/spec-configuration/containerTemplatesOCI.ts

```diff
@@ -36,8 +36,13 @@ export async function fetchTemplate(params: CommonParams, selectedTemplate: Sele
 		output.write(`Failed to fetch template manifest for ${userSelectedId}`, LogLevel.Error);
 		return;
 	}
+	const blobDigest = ociManifest?.manifestObj?.layers[0].digest;
```

Comment:
Can layers be empty?

Reply:
For the custom dev container media types we have out in the wild, there should always be exactly one layer (the tgz or single json collection file).  That said it is good here to validate that.